Automattic/woocommerce-payments
# Hide Link payment method for non-us accounts

Fixes #4668 

#### Changes proposed in this Pull Request
Stripe Link is available only in US. 
This PR updates the available payment methods based, in order to add Link only for US country and if the capability exists for this payment method.








#### Testing instructions





* With an US account, enable Link payment method in admin
* Create an order with Link payment method. 
* No errors should occur
* Test also with Link capability disabled. Link option should not be displayed in Express checkout section
* Link a non US account.
* Go to available payment methods. Link should not be visible. Note: if you previously enabled Link, save the form, so the Link will be removed from the enabled payment methods
* Create an order. No errors should occur

-------------------


**Post merge**
